authentication-service/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+import com.saveourtool.save.buildutils.*
+
+plugins {
+    id("com.saveourtool.save.buildutils.kotlin-jvm-configuration")
+    id("com.saveourtool.save.buildutils.spring-boot-configuration")
+    alias(libs.plugins.download)
+    kotlin("plugin.allopen")
+    alias(libs.plugins.kotlin.plugin.jpa)
+}
+
+kotlin {
+    allOpen {
+        annotation("javax.persistence.Entity")
+        annotation("org.springframework.stereotype.Service")
+    }
+
+    sourceSets {
+        sourceSets.all {
+            languageSettings.optIn("kotlin.RequiresOptIn")
+        }
+    }
+}
+
+dependencies {
+    implementation(projects.saveCloudCommon)
+    implementation(libs.spring.boot.starter.security)
+    implementation(libs.spring.security.core)
+    implementation(libs.spring.jdbc)
```

Comment:
can we add `starter` to field name to show that it's a starter, not a just library

Reply:
Changed